cmd/tailscale/cli/web.html

```diff
@@ -99,13 +99,13 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 		<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
 	</div>
 	{{ if .IsSynology }}
+		{{ if not .TUNMode }}
 		<div class="border border-gray-200 bg-orange-0 rounded-lg p-2 pl-3 pr-3 mb-8 width-full text-orange-800">
-			Outgoing access {{ if true }}enabled{{ else }}not configured{{ end }}.
-			<nobr><a href="https://tailscale.com/kb/1152/synology-outbound/"
-				class="font-medium link"
-				target="_blank"
-				rel="noopener noreferrer">Learn more &rarr;</a></nobr>
+			Outgoing traffic of services is not configured.
```

Comment:
Can we do the wording change in its own PR?

I'm not a huge fan of this but it's fine if we're consistent, but https://tailscale.com/kb/1131/synology/#enabling-synology-outbound-connections doesn't match this